lib/typo_killer/words_parser.ex

```diff
@@ -17,7 +17,12 @@ defmodule TypoKiller.WordsParser do
     |> MapSet.new()
   end
 
-  @spec find_words(file :: String.t()) :: list()
```

Comment:
I think you accidentally deleted this spec here

Reply:
Fixed